packages/react/src/DuotoneText/DuotoneText.module.css

```diff
@@ -0,0 +1,7 @@
+.DuotoneText {
+  color: var(--duotone-color, var(--brand-DuotoneText-text-color-muted));
```

Comment:
Why do we need `--duotone-color`? I can't see a reference to it anywhere. We probably also want to fix this to `var(--brand-DuotoneText-text-color-muted)` anyway?